src/containers/BookDetails.tsx

```diff
@@ -0,0 +1,60 @@
+import React, {useEffect, useState} from 'react'
+import {getBook} from '../external/BooksApiClient';
+import { useParams } from 'react-router';
+import { Book, SingleBookResponse } from '../models/Book';
+import 'antd/dist/antd.css';
+import {Card, Row, Col} from 'antd';
```

Comment:
Not a biggy, just seeing heaps of inconsistent spacing, double returns and no returns between consts/funcs/etc.
eg: 
`import {Card, Row, Col} from 'antd';`
`import { useParams } from 'react-router';`

Suggest installing prettier.